seatunnel-apis/seatunnel-api-flink/src/main/java/org/apache/seatunnel/flink/util/ConfigKeyName.java

```diff
@@ -45,4 +45,9 @@ private ConfigKeyName() {
     public static final String MIN_STATE_RETENTION_TIME = "execution.query.state.min-retention";
     public static final String STATE_BACKEND = "execution.state.backend";
     public static final String PLANNER = "execution.planner";
+    public static final String Metric_Interval = "execution.metrics.interval";
```

Comment:
Use unified name in spark/flink. There config name is for `SeaTunnel`, not only for `Flink` or `Spark`.